src/platform/testing/e2e/megamenu/megamenu.js

```diff
@@ -1,11 +1,7 @@
 const E2eHelpers = require('../helpers');
```

Comment:
I recommend reading this PR in Unified view instead of Split. 

![image](https://user-images.githubusercontent.com/6130520/117874297-653f7d00-b266-11eb-907e-9857d7d6a2a4.png)